library/src/main/java/com/schibsted/spain/barista/interaction/BaristaSeekBarViewInteractions.kt

```diff
@@ -0,0 +1,29 @@
+package com.schibsted.spain.barista.interaction
+
+import android.support.annotation.IdRes
+import android.support.test.espresso.Espresso
+import android.support.test.espresso.matcher.ViewMatchers
+import com.schibsted.spain.barista.interaction.BaristaScrollInteractions.safelyScrollTo
+import com.schibsted.spain.barista.internal.viewaction.SeekBarActions.setProgress
+import com.schibsted.spain.barista.internal.viewaction.SeekBarActions.setProgressToMax
+import com.schibsted.spain.barista.internal.viewaction.SeekBarActions.setProgressToMin
+
+object BaristaSeekBarInteractions {
+  @JvmStatic
+  fun setProgress(@IdRes seekBarId: Int, progress: Int) {
+    safelyScrollTo(seekBarId)
+    Espresso.onView(ViewMatchers.withId(seekBarId)).perform(setProgress(progress))
```

Comment:
I'm thinking... both the BaristaInteraction and the view Action static methods have the same name. Even though one it's an internal method, to the outside is public.
We've had problems with this before (https://github.com/SchibstedSpain/Barista/issues/100).

I'm not sure which method could we change to what. Currently they both make sense :(
Opinions? @jamesbluecrow @rocboronat @alorma @SmasSive?

Reply:
That is a good point, I'm thinking of different alternatives like:

```
- setProgress() vs changeProgressTo() vs swipeProgressTo()
- setProgressToMax() vs changeProgressToMax() vs swipeProgressToMax()
- setProgressToMin() vs changeProgressToMin() vs vs swipeProgressToMin()
```
for some reason I think that I like more the `swipeProgressTo()` pattern for `BaristaSeekBarInteractions` since it represents what it is actually doing, swiping the progress.

Reply:
From @SmasSive comment:
> I'd go for setProgress for the internal method because it sounds more techie (setter and getter) and changeProgress for the public one because it implies not only setting the progress, maybe in a future implies more actions. What do you think?

I'm going to use your argument against you :P 
Since `setProgressTo` is more techie it helps discoverability. In the end Barista users are developers. This name is straightforward, short and simple. No confusion there. Maybe `setProgressTo` fits also.

I have trouble with `swipeProgressTo`. For some reason I see swiping as a less precise action, not the kind of think you do with a SeekBar. I don't see it as much intuitive. BUT that may just be my English level anyways.

To write down a formal opinion, I would go for
- `setProgressTo` for the public methods, because of simplicity
- `setSeekBarProgressTo` for the internal ViewActions, because it exposes implementation a bit.


All these arguments are very thin though. Aw man, why did naming have to be so hard u_u

Reply:
I agree with you here @Sloy , what I was tending to explain is that `setProgress` is very concise, the name says it does just one thing (or should it do) and `swipeProgressTo` something more open that could imply more things, so gathering all the info and opinions... I think in your last comment you nailed it! 🗡

Reply:
SGTM too 👍 
I will update it today!